docs/getting-started/linux_nvidia.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Getting Started with Linux NVidia
+title: Install on Linux NVidia
```

Comment:
Not 100% sure the official way we're supposed to write "NVidia", we should confirm this so it's brand correct.

Reply:
So I believe the brand offical way to write it is NVIDIA.